docs/fastdds/logging/logging_filtering.rst

```diff
@@ -108,4 +108,4 @@ The logging module's filters can be reset with member function |Log::Reset-api|.
     * Disabling :ref:`dds_layer_log_file_context` component.
     * Enabling :ref:`dds_layer_log_function_name` component.
     * Clear all :ref:`dds_layer_log_filter`.
-    * Clear all consumers.
+    * Clear all consumers and set a STDOUT or STDOUTERR consumer (depending on the CMake option |LOG_CONSUMER_DEFAULT|).
```

Comment:
I'm wondering it this was even true to begin with

Reply:
In order to clear the filters, `Log::Reset()` must be called and this method performs all those operations.